### PR TITLE
Provide default implementation for IHubFilter.InvokeMethodAsync

### PR DESCRIPTION
**Pull request by BrennanConroy**
@halter73 mentioned we should probably add a default for this as well. It's not too hard to imagine filters being written that don't want to implement `InvokeMethodAsync`.

### CODE REVIEW

**Review by halter73 — APPROVED** [review]

File: src/SignalR/server/Core/src/IHubFilter.cs [hunk 1]
```diff
@@ -19,7 +19,7 @@ public interface IHubFilter
         /// <param name="invocationContext">The context for the method invocation that holds all the important information about the invoke.</param>
         /// <param name="next">The next filter to run, and for the final one, the Hub invocation.</param>
         /// <returns>Returns the result of the Hub method invoke.</returns>
-        ValueTask<object> InvokeMethodAsync(HubInvocationContext invocationContext, Func<HubInvocationContext, ValueTask<object>> next);
+        ValueTask<object> InvokeMethodAsync(HubInvocationContext invocationContext, Func<HubInvocationContext, ValueTask<object>> next) => next(invocationContext);
```
Comment (halter73) [inline src/SignalR/server/Core/src/IHubFilter.cs:22]: It might be worth adding a simple test that verifies you don't have to implement InvokeMethodAsync.

Comment (JamesNK) [inline src/SignalR/server/Core/src/IHubFilter.cs:22]: Why are hub filters an interface with a default implementation rather than an abstract base class?

Comment (davidfowl) [reply src/SignalR/server/Core/src/IHubFilter.cs:22]: We're rebels. I actually think it's a good experiment to try this on

Comment (BrennanConroy) [reply src/SignalR/server/Core/src/IHubFilter.cs:22]: I'm not a fan of how people generally call `base.FunctionName();` (and intellisense adds it automatically) with abstract base classes when the base class does nothing

**Pull request merged**